src/openrct2/scenario/ScenarioRepository.cpp

```diff
@@ -460,7 +460,7 @@ class ScenarioRepository final : public IScenarioRepository
                 // Get scenario using RCTC style name of RCT2 scenario
                 scenario = GetByFilename((scenarioBaseName + ".sea").c_str());
             }
-            // gScenarioFileName .Park scenarios is the full file path instead of just <scenarioName.park>, so need to convert
+            // ScenarioFileName .Park scenarios is the full file path instead of just <scenarioName.park>, so need to convert
```

Comment:
```suggestion
            // GameState_t::ScenarioFileName .Park scenarios is the full file path instead of just <scenarioName.park>, so need to convert
```